.github/workflows/openmetadata-ingestion-core-version-comment.yml

```diff
@@ -20,6 +20,9 @@ on:
     paths:
     - 'catalog-rest-service/src/main/resources/json/schema/**'
 
+concurrency: 
+  group: ${{ github.head_ref }}
```

Comment:
let's update the group name to be {workflow_name}-{github_head_ref}.

Reply:
I have updated as per the workflow name